infra/gp-cluster-monitoring-config/templates/00-cluster-monitoring-config.yaml

```diff
@@ -11,6 +11,9 @@ data:
       {{- include "infranodes.enabled" . | nindent 6 }}
     prometheusK8s:
       {{- include "infranodes.enabled" . | nindent 6 }}
+      externalLabels:
+        cluster: {{. Values.}}
+        prometheus: user_workfload
```

Comment:
https://docs.openshift.com/container-platform/4.10/monitoring/configuring-the-monitoring-stack.html#attaching-additional-labels-to-your-time-series-and-alerts_configuring-the-monitoring-stack :
Do not use prometheus or prometheus_replica as key names, because they are reserved and will be overwritten.

auch ein typo drin. und wahrscheinlich eher nur: openshift_workload?